### PR TITLE
Add allow_content_tools and force_intger_mv to AV1 enc

### PR DESCRIPTION
**Pull request by rosetta-jpn**

### CODE REVIEW

**Comment by rosetta-jpn** [comment]
@XinfengZhang

**Review by XinfengZhang — APPROVED** [review]
LGTM

**Comment by XinfengZhang** [comment]
@leyu-yao

**Comment by XinfengZhang** [comment]
commit message integer?

File: va/va_enc_av1.h [hunk 1]
```diff
@@ -656,8 +656,12 @@ typedef struct  _VAEncPictureParameterBufferAV1 {
              * Otherwise disable palette encoding.
              */
             uint32_t    palette_mode_enable             : 1;
+            /** \brief Corresponds to AV1 syntax element of the same name. */
```
Comment (XinfengZhang) [inline va/va_enc_av1.h:659]: alignment

**Review by leyu-yao — APPROVED** [review]
LGTM

**Comment by rosetta-jpn** [comment]
Updated!

**Comment by rosetta-jpn** [comment]
Gentle ping.

**Review by cchunbo — APPROVED** [review]

**Pull request merged**